homeassistant/components/ombi/__init__.py

```diff
@@ -0,0 +1,96 @@
+"""Support for Ombi."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+)
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_URLBASE, DEFAULT_PORT, DEFAULT_SSL, DEFAULT_URLBASE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_API_KEY): cv.string,
+                vol.Required(CONF_HOST): cv.string,
+                vol.Required(CONF_USERNAME): cv.string,
+                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+                vol.Optional(CONF_URLBASE, default=DEFAULT_URLBASE): cv.string,
+                vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+
+def setup(hass, config):
+    """Set up the Ombi component platform."""
+    import pyombi
+
+    urlbase = f"{config[DOMAIN][CONF_URLBASE].strip('/') if config[DOMAIN][CONF_URLBASE] else ''}/"
```

Comment:
This option has a default.

Reply:
Please add a custom validator that strips the slash from the url. We want all config modifications to happen in the config schema. See `config_validation.py` for examples.